ui/packages/app/src/components/Accordion/Accordion.svelte

```diff
@@ -14,14 +14,39 @@
 
 <div
 	id={elem_id}
-	class="p-3 border border-gray-200 dark:border-gray-700 rounded-lg flex flex-col gap-3 hover:border-gray-300 dark:hover:border-gray-600 transition"
-	class:hidden={!visible}
+	class="container border-gray-200 dark:border-gray-700 hover:border-gray-300 dark:hover:border-gray-600 "
+	style:display={visible ? "block" : "none"}
 >
-	<div on:click={toggle} class="w-full flex justify-between cursor-pointer">
+	<div on:click={toggle} class="label-wrap">
 		<span>{label}</span>
-		<span class:rotate-90={!_open} class="transition">▼</span>
+		<span style:transform={_open ? "rotate(0)" : "rotate(90deg)"} class="icon"
+			>▼</span
+		>
 	</div>
 	<Column visible={_open}>
 		<slot />
 	</Column>
 </div>
+
+<style>
+	.container {
+		display: flex;
+		flex-direction: column;
+		gap: var(--size-3);
+		padding: var(--size-3);
+		border-width: 1px;
+		transition: 150ms;
+		border-radius: var(--radius-lg);
+	}
+
+	.label-wrap {
```

Comment:
In Model3D the `x` in the input model3d component is smaller

### This branch
![image](https://user-images.githubusercontent.com/41651716/209231451-111f4662-a986-40a7-83cb-060ad1f0db30.png)

### 3.15.0
![image](https://user-images.githubusercontent.com/41651716/209231417-fa4b75e4-eba6-4c7f-b5b6-e9dc3ef30a67.png)


Also the file output component doesn't have a scrollbar anymore

### This branch
![image](https://user-images.githubusercontent.com/41651716/209231532-93c21768-8d9c-4c74-9d66-9e75cd239948.png)

### 3.15.0
![image](https://user-images.githubusercontent.com/41651716/209231572-77694eb3-1363-4f75-8368-7e6581d48422.png)